simulation.py

```diff
@@ -1022,9 +1073,12 @@ def take_average(self):
         if self.is_ab_test:
             for condition in ['control', 'treatment']:
                 self.add_stats(f"avg_degree_{condition}")
-                self.add_stats(f"degree_variance_{condition}")
                 self.add_stats(f"global_clustering_{condition}")
                 self.add_stats(f"gini_coeff_{condition}")
+                if condition == 'control':
+                    self.add_stats(f"avg_degree_{condition}_adjusted")
+                    self.add_stats(f"global_clustering_{condition}_adjusted")
+                    self.add_stats(f"gini_coeff_{condition}_adjusted")
```

Comment:
seems like we are repeating the same code pattern a lot; might be worthwhile at some point to address

Reply:
yeah for sure